presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -538,6 +547,45 @@ protected Scope visitCreateView(CreateView node, Optional<Scope> scope)
             return createAndAssignScope(node, scope);
         }
 
+        @Override
+        protected Scope visitCreateFunction(CreateFunction node, Optional<Scope> scope)
+        {
+            analysis.setUpdateType("CREATE FUNCTION");
+
+            // Check parameter
+            List<String> duplicateParameters = node.getParameters().stream()
```

Comment:
I need the duplicate parameter names to construct a user-friendly error message, but size checking won't tell me what are duplicated.

Alternatively, printing one name instead of a list is fine with me as well, but that does not make the code simpler.

Reply:
I'd just print the whole parameter list lol but sure.